src/mpi/comm/comm_split_type.c

```diff
@@ -27,21 +27,43 @@ int MPI_Comm_split_type(MPI_Comm comm, int split_type, int key, MPI_Info info, M
 #define MPI_Comm_split_type PMPI_Comm_split_type
 
 #undef FUNCNAME
-#define FUNCNAME MPIR_Comm_split_type_impl
+#define FUNCNAME MPIR_Comm_split_type
 #undef FCNAME
 #define FCNAME MPL_QUOTE(FUNCNAME)
-int MPIR_Comm_split_type_impl(MPIR_Comm * comm_ptr, int split_type, int key,
-                              MPIR_Info * info_ptr, MPIR_Comm ** newcomm_ptr)
+int MPIR_Comm_split_type_shmem_node(MPIR_Comm * comm_ptr, int key, MPIR_Comm ** newcomm_ptr)
```

Comment:
Why are `FUNCNAME` and the actual function name different?

Reply:
Fixed.